.github/workflows/build.yml

```diff
@@ -127,8 +127,9 @@ jobs:
             libxml2-dev
             # spellcheck
             libenchant-dev
+            ä multiterm
```

Comment:
Was this meant to be `# multiterm` ?

Reply:
Yeah, typo on German keyboard with tired eyes :).